internal/clusterconfig/clusterconfig.go

```diff
@@ -47,12 +47,18 @@ const (
 	idmsFileName  = "image-digest-mirror-set.json"
 	icspsFileName = "image-content-source-policy-list.json"
 
-	caBundleCMName            = "user-ca-bundle"
-	caBundleFileName          = caBundleCMName + ".json"
-	catalogueSourcesNamespace = "openshift-marketplace"
-	catalogueSourcesFileName  = "catalogue-sources-list.json"
+	caBundleCMName   = "user-ca-bundle"
+	caBundleFileName = caBundleCMName + ".json"
+
+	networkDir = "/opt/openshift/network-configuration"
 )
 
+var hostPath = common.Host
+var listOfNetworkFilesPaths = []string{
+	"/etc/hostname",
+	"/etc/NetworkManager/system-connections",
+}
+
 var (
```

Comment:
Consider moving the new vars in the var() block

Reply:
right, missed it